v23.1/security-reference/transport-layer-security.md

```diff
@@ -23,8 +22,7 @@ This page provides a conceptual overview of Transport Layer Security (TLS) and t
 **Learn more:**
 
 - [Manage PKI certificates for a CockroachDB deployment with HashiCorp Vault](../manage-certs-vault.html)
-- [Use the CockroachDB CLI to provision a development cluster](../manage-certs-cli.html)
-
+- [CockroachDB Cloud: Certificate Authentication for SQL Clients](../../cockroachcloud/client-certs-dedicated.html)
```

Comment:
Same feedback about CockroachDB Cloud in the link text.

Reply:
same response